cli/tauri.js/bin/tauri.js

```diff
@@ -32,24 +34,27 @@ const tauri = function (command) {
     command === '--help' ||
     command === 'help'
   ) {
-    console.log(`
-    Description
-      This is the Tauri CLI.
-    Usage
-      $ tauri ${cmds.join('|')}
-    Options
-      --help, -h     Displays this message
-      --version, -v  Displays the Tauri CLI version
-    `)
+    
+    console.log(chalk.cyan(figlet.textSync('Tauri')));
+    console.log(chalk.green('Description'))
+    console.log('This is the Tauri CLI');
+    console.log(chalk.magenta('Usage'));
+    console.log('$ tauri ${cmds.join('|')}')
+    console.log(chalk.blue('Options'))
+    console.log('--help, -h     Displays this message \n  --version, -v  Displays the Tauri CLI version');
     process.exit(0)
     // eslint-disable-next-line no-unreachable
     return false // do this for node consumers and tests
   }
 
   if (command === '-v' || command === '--version') {
-    console.log(require('../package.json').version)
+    console.log(chalk.green(`${package.version}`));
+    // also added colors to beautify the cli
     return false // do this for node consumers and tests
   }
+  if (command ==='-no-update-notifier || command === '--no-update-notifier'){
+      const noUpdates = true;
+      }
```

Comment:
there is a missing single quote
and I think it should be one flag like this and better name for the variable
```js 
let shouldNotify = true; // the variable must be defined here, otherwise it is inaccessible by other code. 
if (command === '--no-update-notifier') {
	shouldNotify = false;
}
```

Also I think  we should not provide this flag anyway and always show the update notification if needed.